analyses/tp53_nf1_score/10-tp53-tumor-purity-threshold.Rmd

```diff
@@ -0,0 +1,590 @@
+    y = "TP53 score"
+  ) +
+  theme(
+    axis.text.x = element_text(hjust = 1, angle = 30, size = 7)
+  )
+
+tp53_violins
+```
+
+- Previously highest median groups are all still high (DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas).
+- Previously lowest median groups are _mostly_ still low (LGGs, meningiomas, and schwannomas) but Gangliomas now show a higher median value. 
+These scores also have a very broad spread, and their overall distribution appears very similar to that previously observed.
+
+As such, these results are broadly consistent.
+
+
+Now we'll export `tp53_violins`:
+
+```{r}
+ggsave(
+  tp53_cancer_groups_pdf,
+  tp53_violins,
+  width = 8, 
+  height = 3.75
+)
+
+```
+
+## Re-analysis 5: LFS patients
+
+
+In the manuscript, we found:
+
+https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L203
+
+> Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations (**Tables S1 and S3**).
+
+```{r}
+status_original %>%
+  tidyr::drop_na(tp53_score) %>%
+  dplyr::mutate(lfs = cancer_predispositions == "Li-Fraumeni syndrome") %>%
+  dplyr::group_by(lfs) %>%
+  dplyr::summarize(n(), 
+                   median(tp53_score))
+```
+
+Here we see there are 9 LFS samples, so it's unclear where N=8 is from.
+This should be revisited.
+
+Now, we observe:
+
+```{r}
+status_thresholded %>%
+  tidyr::drop_na(tp53_score) %>%
+  dplyr::mutate(lfs = cancer_predispositions == "Li-Fraumeni syndrome") %>%
+  dplyr::group_by(lfs) %>%
+  dplyr::summarize(n(),
+                   median(tp53_score))
+```
+
+Results are consistent - after thresholding, LFS samples still have higher scores compared to non-LFS samples.
+
+
+## Re-analysis 6: Hypermutators 
+
+In the manuscript, we found: 
+
+https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L227
+
+> With one exception, hypermutant and ultra-hypermutant tumors had high _TP53_ scores (> 0.5) and telomerase activity.
+
+To confirm the above, we'll first check the original TP53 scores of these samples.
+
+First, figure out which samples these are:
+
+```{r}
+# These samples were identified as coming from hypermutators in the manuscript: 
+# https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L232-L247
+# These IDs are WGS, but do not include cell lines.
+hypermutator_bs_ids <- c("BS_85Q5P8GF", "BS_P0QJ1QAH", "BS_20TBZG09", "BS_8AY2GM4G", "BS_02YBZSBY", "BS_VW4XN9Y7", "BS_F0GNWEJJ", "BS_P3PF53V8")
+
+# Get the sample ids
+original_sample_ids <- metadata %>%
+  dplyr::filter(Kids_First_Biospecimen_ID %in% hypermutator_bs_ids) %>%
+  dplyr::pull(sample_id) %>%
+  unique() 
+
+status_original %>%
+  dplyr::filter(sample_id %in% original_sample_ids) %>%
+  dplyr::pull(tp53_score)
```

Comment:
Are there multiple RNA-seq biospecimens that map to the same sample ID and that's accounts for the other < 0.5 score? (If this is true, we probably want to still change the language in the manuscript.)